src/libraries/System.Collections/src/System/Collections/Generic/PriorityQueue.cs

```diff
@@ -479,7 +439,7 @@ private void RemoveRootNode()
         /// <summary>
         /// Gets the index of the first child of an element.
         /// </summary>
-        private int GetFirstChildIndex(int index) => Arity * index + 1;
+        private int GetFirstChildIndex(int index) => (index << Log2Arity) + 1;
```

Comment:
I assume this is just for consistency with GetParentIndex above?

Reply:
That's right.